detection_rules/schemas/definitions.py

```diff
@@ -51,7 +51,7 @@
 Severity = Literal['low', 'medium', 'high', 'critical']
 Sha256 = NewType('Sha256', str, validate=validate.Regexp(SHA256_PATTERN))
 UUIDString = NewType('UUIDString', str, validate=validate.Regexp(UUID_PATTERN))
-Maturity = Literal['development', 'experimental', 'beta', 'production', 'deprecated']
+Maturity = Literal['development', 'experimental', 'beta', 'production', 'deprecated', 'hunt']
```

Comment:
@brokensound77 added hunt as an option.

Reply:
hmm I think once a rule goes production, it either should stay in production or go to deprecated. Otherwise, we just create another edge case that we have to handle somewhere else unnecessarily

I think "hunt" isn't a property of maturity, but more about the expected use case

Reply:
I updated the PR to `deprecated` maturity